spyder/plugins/editor/widgets/base.py

```diff
@@ -380,6 +380,24 @@ def keyPressEvent(self, event):
             super(TextEditBaseWidget, self).keyPressEvent(event)
 
     #------Text: get, set, ...
+    def add_to_cell_list(self, oedata):
+        """Add new cell to cell list."""
+        self._cell_list.append(oedata)

```

Comment:
What happens when cells are removed? is this also being updated somewhere or we just keep accumulating information on this variable?

Reply:
Line 394 filters the list every call

Reply:
👍